docs/modules/ROOT/pages/vshn-managed/postgresql/usage.adoc

```diff
@@ -5,32 +5,176 @@ The latter references the `postgres-creds` secret that will contain the access k
 
 [source,yaml]
 ----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: widera-testing
+  labels:
+    name: widera-testing
+---
 apiVersion: vshn.appcat.vshn.io/v1
 kind: VSHNPostgreSQL
 metadata:
   name: my-postgres-example
-  namespace: my-namespace
+  namespace: widera-testing
 spec:
   parameters:
     service:
       majorVersion: "15"
+    backup:
+      schedule: '0 22 * * *'
   writeConnectionSecretToRef:
     name: postgres-creds-connection
 ---
 apiVersion: v1
 kind: Pod
 metadata:
   name: postgres-client
+  namespace: widera-testing
 spec:
   containers:
   - name: postgres
     image: postgres:15
+    resources:
+      limits:
+        memory: "128Mi"
+        cpu: "500m"
     command:
       - /bin/sh
       - -c
     args:
-      - PGPASSWORD=$POSTGRESQL_PASSWORD psql -h $POSTGRESQL_HOST -d $POSTGRESQL_DB -U $POSTGRESQL_USER -l
+      - PGPASSWORD=$POSTGRESQL_PASSWORD psql "sslmode=verify-ca sslrootcert=/etc/secret-volume/ca.crt host=$POSTGRESQL_HOST port=$POSTGRESQL_PORT dbname=$POSTGRESQL_DB" -U $POSTGRESQL_USER -l
     envFrom:
       - secretRef:
           name: postgres-creds-connection
-  restartPolicy: Never
+    volumeMounts:
+    - name: secret-volume
+      readOnly: true
+      mountPath: "/etc/secret-volume"
+  volumes:
+  - name: secret-volume
+    secret:
+      defaultMode: 0600
+      secretName: postgres-creds-connection
+  restartPolicy: OnFailure
+----
+
+== Debug instance by Yourself
+
+You can easily check what happend during creation by checking status fields. PostgreSQL managed by VSHN is composed from many resources, responsible for
```

Comment:
```suggestion
You can easily check what happened during creation by checking the status fields. PostgreSQL by VSHN is composed from many resources, responsible for
```

Please do not insert a newline in the middle of a sentence. AsciiDoc is meant to be readable in the source as well.